### ISSUE TITLE
'

### ISSUE THREAD

**Issue by tonydisera** [tonydisera/gene.iobio#374]

**Comment by tonydisera** [comment]
<img width="1440" alt="screenshot 2016-07-25 20 10 46" src="https://cloud.githubusercontent.com/assets/8420103/17123877/225ce954-52a4-11e6-8db6-54b7cba4f66d.png">

**Comment by AlistairNWard** [comment]
I'm having a problem with this. If I copy and paste the list of genes, and the click Done, I'm not getting any badges appear.

**Comment by AlistairNWard** [comment]
![screen shot 2016-07-26 at 9 01 20 am](https://cloud.githubusercontent.com/assets/523111/17138740/ae507050-530f-11e6-816b-e1ae81458b95.png)

**Comment by AlistairNWard** [comment]
If I get time, I might start looking at Selenium again and use this as the first test. If we input genes, do we get badges. Right now it would fail, so would work as a test. This is commit 779fc6f80803b99e5e11b9b0bc2ec51da7bde54d, so hopefully I can recreate the fail on this commit.

**Comment by AlistairNWard** [comment]
The URL is populated and hitting refresh will pull in the genes, but not clicking Done.

**Comment by tonydisera** [comment]
I will look at this right now.

**Comment by tonydisera** [comment]
Hi Al.  Give it another try.  I just fixed the bug that was causing the gene badge container to be deleted.

**Comment by AlistairNWard** [comment]
Yup, all fixed. If you checkout the commit above, the error can be regenerated, so I'm going to reassign this to me and take a look at using this as a selenium test.

**Comment by AlistairNWard** [comment]
Selenium test included in #376, so this issue is just about the analyze all getting stuck.

**Comment by AlistairNWard** [comment]
Analyze all no longer getting stuck. Check.
